vendor/src/github.com/bsphere/le_go/certs.go

```diff
@@ -0,0 +1,222 @@
+package le_go
+
+// Certs from https://logentries.com/doc/certificates
+var pemCerts = []byte(`
```

Comment:
@cpuguy83 yes, it was added by @StephenHynes7 ... maybe we can just use system cert pool, I'll try to do that.